solr/docker/README.md

```diff
@@ -0,0 +1,49 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Solr Docker
+----
+
+This Solr docker module creates both a local Docker image from the source as well as the official Solr Dockerfile.
+This allows for local images to be completely compatible with the official Solr images available on DockerHub.
+
+In order to build/test/tag your Docker images using local Solr source code, please refer to `./gradlew helpDocker`
```

Comment:
```suggestion
In order to build/test/tag your Docker images using local Solr source code, please refer to `./gradlew helpDocker` in a git checkout or a source-release download.
```
As I understand, this README is copied into the binary distro.

Can it be confusing for a user untar'ing the release that the README talks about building from source and gradlew? I suppose most users know that by "source" we mean the git checkout or the source tgz download, but should we call it out in some way?